src/routes/ExercisesScreens.tsx

```diff
@@ -26,16 +40,22 @@ interface ExercisesScreenProps {
 const ExercisesScreen = ({ route, navigation }: ExercisesScreenProps): JSX.Element => {
   const { discipline } = route.params
   const { title } = discipline
+  const [isModalVisible, setIsModalVisible] = useState<boolean>(false)
 
   const Header = <Title title={title} description={childrenDescription(discipline)} />
 
   const handleNavigation = (item: Exercise): void => {
-    if (item.title === labels.exercises.wordChoice.title && discipline.numberOfChildren < MIN_WORDS) {
-      Alert.alert(labels.exercises.wordChoice.errorWrongModuleSize)
+    const CurrentLevel = 4 //  here is a hard coded level for locking exercice modal
```

Comment:
naming of variables is always in camelCase, so the first letter should be lower case.